android/engine/src/main/java/org/smartregister/fhircore/engine/auth/AccountAuthenticator.kt

```diff
@@ -170,6 +174,18 @@ constructor(
 
   fun getUserInfo(): Call<ResponseBody> = oAuthService.userInfo()
 
+  // TODO move to some external file
+  suspend fun getPractitionerDetails(keycloak_uuid: String): org.hl7.fhir.r4.model.Bundle {
```

Comment:
if it is deployed on server plz implement.

Reply:
It's not deployed yet.

Reply:
Now deployed